src/lib/stack-client.js

```diff
@@ -0,0 +1,236 @@
+/* global __TARGET__ */
+/* eslint-env browser */
+
+import CozyClient from 'cozy-client'
+import initializeRealtime from './realtime'
+
+import {
+  ForbiddenException,
+  ServerErrorException,
+  NotFoundException,
+  MethodNotAllowedException,
+  UnavailableStackException,
+  UnavailableSettingsException,
+  UnauthorizedStackException
+} from './exceptions'
+
+import getIcon from './icon'
+
+let cozyClient = undefined
+
+const errorStatuses = {
+  '401': UnauthorizedStackException,
+  '403': ForbiddenException,
+  '404': NotFoundException,
+  '405': MethodNotAllowedException,
+  '500': ServerErrorException
+}
+
+// Utilizy to fetch the resources with cozy-client
+// maintening the compatibility with the legacy 
+// standalone cozy-bar client
+const fetchJSON = function(method, path, body, options={}) {
+  
+  // We mirror here a few lines from cozy-stack-client
+  // because we want a customized fetchJSON
+  const headers = (options.headers = options.headers || {})
+  headers['Accept'] = 'application/json'
+  if (method !== 'GET' && method !== 'HEAD' && body !== undefined) {
+    if (!headers['Content-Type']) {
+      headers['Content-Type'] = 'application/json'
+      body = JSON.stringify(body)
+    }
+  }
+
+  cozyClient.getStackClient().fetch(method, path, body, options).then(
+    resp => {
+      if (typeof errorStatuses[resp.status] === 'function') {
+        throw new errorStatuses[resp.status]()
+      }    
+      const contentType = resp.headers.get('content-type')
+      const isJson = contentType && contentType.indexOf('json') >= 0
```

Comment:
Nit: `contentType.includes('json')` is more readable IMHO (it works everywhere except in IE)

Reply:
✅ it was taken from the legacy code and not rewritten, I'll do that